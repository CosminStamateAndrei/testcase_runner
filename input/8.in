3 3 2
-9 4 7 
3 -5 3 
-1 5 -10 
-2 -1 
-5 -6 
-1 2 
