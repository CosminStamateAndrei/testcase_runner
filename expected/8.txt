Matrix A
  -9    4    7 
   3   -5    3 
  -1    5  -10 
Matrix B
  -2   -1 
  -5   -6 
  -1    2 
Matrix C
  -9   -1 
  16   33 
 -13  -49 
